pkgs/development/compilers/zig/0.10.nix

```diff
@@ -8,15 +8,15 @@
 , zlib
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
Given that I am refactoring this (#242153), you can remove Zig itself for now.

Reply:
> EDIT: better wait the conflict.

So should I do anything? Or once it's merged you rebase your commits on master then deal with the conflict?